app/src/main/java/com/kickstarter/ui/activities/ProjectPageActivity.kt

```diff
@@ -264,17 +264,19 @@ class ProjectPageActivity :
             .subscribe { showPledgeFragment(it) }
 
         this.viewModel.outputs.startRootCommentsActivity()
+            .map { it.second }
```

Comment:
Could you move that `.map` to the ViewModel, and emit to the output only the projectData? same for line 275